package-local-nicknames-tests.lisp

```diff
@@ -131,6 +140,15 @@
   (let ((*package* (find-package :package-local-nicknames-test-1)))
     (assert (not (find-package :l)))))
 
+(define-test test-package-local-nicknames-nickname-removal-char
+  (declare (optimize (debug 3) (speed 0)))
```

Comment:
Is this declaration required? If so, shouldn't it occur in all tests? Also, if so, what will happen if code without this declaration is generated?